temperature/crud.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from temperature import models, schemas
+from city import models as city_models
```

Comment:
[change request] Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)